### PR TITLE
Solves issues 18 and 23

### PR DESCRIPTION
**Pull request by stephanenicolas**
Solves issues:
* #18 
* #23 

I had to remove the jacoco plugin as it had not been updated to gradle 5.x

### CODE REVIEW

File: feature-adapter-sample-rx/build.gradle [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 26
-    buildToolsVersion "26.0.2"
+    compileSdkVersion 28
+    buildToolsVersion "28.0.3"
```
Comment (ww2lin) [inline feature-adapter-sample-rx/build.gradle]: do we need to change the targetSdkVersion ?

**Comment by ww2lin** [comment]
one comment otherwise good to merge.

File: feature-adapter-sample-rx/build.gradle (hunk 1 above)
Comment (ww2lin) [reply feature-adapter-sample-rx/build.gradle]: talked offline, this is not needed.

**Comment by stephanenicolas** [comment]
After talking withh @ww2lin , we agreed we don't need to change the target sdk for now.

**Pull request merged**